src/styles/_mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin onTablet {
+  @media (min-width:744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width:1024px) {
```

Comment:
Consider using a variable for the desktop breakpoint value (e.g., $desktop-breakpoint) to make it easier to maintain and update in the future. 🖥️